core/src/main/java/io/camunda/migrator/config/MigratorProperties.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.migrator.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(MigratorProperties.PREFIX)
+public class MigratorProperties {
```

Comment:
Whoops. I forgot to add a configuration class for other Data Migrator config properties. Can we add them now?